### PR TITLE
Add cross-origin resource policy compat data

### PR DESCRIPTION
**Pull request by lol768**
Refs: https://developer.apple.com/documentation/safari_release_notes/safari_12_release_notes https://www.chromestatus.com/feature/4647328103268352 https://webkit.org/blog/8332/release-notes-for-safari-technology-preview-59/

This PR adds compatibility data for the newish Cross-Origin-Resource-Policy header to prevent cross-origin/cross-site reads.

### CODE REVIEW

**Comment by lol768** [comment]
Unsure why Travis dislikes the Safari version.

Also, I suppose we should use https://webkit.org/blog/8296/release-notes-for-safari-technology-preview-56/ (where `From-Origin` was implemented) rather than v59.

**Comment by vinyldarkscratch** [comment]
Looks like you're basing the Safari version based upon the Technology Preview version, not the official Safari releases.  Comparing the versions by date, "12" would probably be the closest version number to what you're looking for.  Otherwise, looking good!

**Comment by lol768** [comment]
>Looks like you're basing the Safari version based upon the Technology Preview version, not the official Safari releases

Ah, I see - it's a confusing versioning scheme... I've amended the commit :+1:

**Comment by Elchi3** [comment]
I don't know if it would be 12. See also https://github.com/mdn/browser-compat-data/issues/2991

File: http/headers/cross-origin-resource-policy.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "http": {
+    "headers": {
+      "corb": {
```
Comment (Elchi3) [inline http/headers/cross-origin-resource-policy.json]:
```suggestion
      "Cross-Origin-Resource-Policy": {
```

File: http/headers/cross-origin-resource-policy.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "73"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": false,
```
Comment (Elchi3) [inline http/headers/cross-origin-resource-policy.json]: Isn't it being standardized?

Comment (lol768) [reply http/headers/cross-origin-resource-policy.json]: I couldn't find an IETF RFC or W3C draft or standard, as I've seen in the past `SameSite` cookie & CSP. I must admit I'm not sure what WhatWG are really doing nor am I up to date with whether W3C is really relevant anymore.. it's in WhatWG's (unversioned?) "living standard" - does that qualify for `"standard_track": true`?.

**Comment by lol768** [comment]
Hey @Elchi3 

>Thanks for your PR and welcome to mdn-browser-compat-data! +1

Thanks for the welcome and review :smiley: 

I've changed the review code that you commented on. Unsure what to do about Safari, part of me thinks it would make sense to have a separate TP variant browser which supports the technology preview versions. Is there a v13 I should use in the meantime?

**Pull request merged**